src/components/Layout/components/Footer/components/LinkList.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyle = makeStyles({
```

Comment:
```suggestion
const useStyles = makeStyles({
```
И не только тут, повторяться не буду )